bin/pycbc_multi_inspiral

```diff
@@ -86,89 +110,254 @@ psd.insert_psd_option_group_multi_ifo(parser)
 scheme.insert_processing_option_group(parser)
 fft.insert_fft_option_group(parser)
 
+#Adding these to see if it lets me use the gating options
+from pycbc.vetoes.sgchisq import SingleDetSGChisq
+pycbc.opt.insert_optimization_option_group(parser)
+pycbc.weave.insert_weave_option_group(parser)
+pycbc.inject.insert_injfilterrejector_option_group(parser)
+SingleDetSGChisq.insert_option_group(parser)
 opt = parser.parse_args()
 
+generate_histograms = False
+generate_snr_plots = False
+generate_chi2_plots = False
+# Polarisation doesn't matter for coherent matched filter so set to zero.
+pol = 0
+# Use the time in the middle of the segment to calculate the antenna patterns.
+t_gps = (opt.trig_end_time[0] + opt.trig_start_time[0])/2
+
+# Set sky position being searched.
+ra = opt.longitude 
+dec = opt.latitude
+
+# Put the ifos in alphabetical order so they are always called in
+# the same order.
+opt.instruments.sort()
+
 strain.verify_strain_options_multi_ifo(opt, parser, opt.instruments)
 strain.StrainSegments.verify_segment_options_multi_ifo(opt, parser,
                                                        opt.instruments)
 psd.verify_psd_options_multi_ifo(opt, parser, opt.instruments)
 scheme.verify_processing_options(opt, parser)
 fft.verify_fft_options(opt,parser)
 
-if opt.verbose:
-    log_level = logging.DEBUG
-else:
-    log_level = logging.WARN
+log_level = logging.DEBUG if opt.verbose else logging.INFO
 logging.basicConfig(format='%(asctime)s : %(message)s', level=log_level)
 
 ctx = scheme.from_cli(opt)
 
+def get_weighted_antenna_patterns(Fp_dict,Fc_dict,sigma_dict):
+    """
```

Comment:
The function documentation here and below doesn't follow the numpy format that we use in PyCBC.

I'm not going to insist on this while these functions are in a executable, but at some point these will make sense to be moved to a `grb_utils.py` module or similar, and then this *will* need to be fixed.